### ISSUE TITLE
CS3: Some characters have wrong number of orbment line in t_orb.tbl

### ISSUE THREAD

**Issue by nnguyen259** [AdmiralCurtiss/SenPatcher#61]
For BaseList entries in t_orb.tbl, the first 2 bytes are for the character_id, then the next 2 bytes are most likely the number of orbment line that character has, which matches up for majority of characters except for a few. Alisa is the first char with this issue. 

Doesn't look like it affects gameplay, but consistency would be nice to have. This was found when I parsed the file trying to convert it into csv.

**Comment by AdmiralCurtiss** [comment]
Noted, though I wonder if this should really be fixed if it has no apparent impact on gameplay.

**Comment by nnguyen259** [comment]
Yeah it's along the line of consistency fix. Something else I found while messing around with that file though, if you give Towa and Patrick some line data other than the default, even if it has the correct syntax, it crashes the game on start up, you couldn't even get to the title screen.
